include/flatbuffers/vector.h

```diff
@@ -116,9 +117,12 @@ template<typename T, typename IT> struct VectorIterator {
   }
 
  private:
-  const uint8_t *data_;
+  Data data_;
 };
 
+template<typename T, typename IT>
+using VectorConstIterator = VectorIterator<T, IT, const uint8_t *>;
```

Comment:
Shouldn't this be in terms of `Data`?

Reply:
Sorry, I don't see the point here.

Reply:
Sorry, i thought it was still part of the VectorIterator struct defined above with `Data`. I see now it is free standing.